training/benchmarks/mixtral_7b/megatron/megatron_main.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+# Runs Mixtral 8x3.5B model on 8 A100 GPUs
+
+DATADIR=$1
+GPUS_PER_NODE=$2
+NNODES=$3
+NODE_RANK=$4
+MASTER_ADDR=$5
+MASTER_PORT=$6
+TP=$7
```

Comment:
TPPPEP should be listed as modifiable

all add globalbs, localbs. rules should be listed at nvidia/mixtral-7b-megatron/README.md